webpack/components/extensions/HostDetails/Tabs/RepositorySetsTab/RepositorySetsTab.js

```diff
@@ -225,6 +225,8 @@ const RepositorySetsTab = () => {
                 toggleGroup,
                 }
           }
+          activeFilters={[toggleGroupState]}
+          defaultFilters={[defaultToggleGroupState]}
```

Comment:
TableWrapper has [logic](https://github.com/jeremylenz/katello/blob/219c364525c0f99db389a52831682d7cd18a21d7/webpack/components/Table/TableWrapper.js#L94) that checks whether active filters have changed, and if so resets you to page 1. I neglected to add activeFilters here, so that was the source of the bug you found.

Reply:
side note, Usually the `defaultFilters` are just hard-coded strings. But here they are dynamic, because the default toggle group state changes depending on if it's a library host or not.